feeds/cache.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Cache
+parent: Feeds
+nav_order: 3
+---
+
+# Central Feed Cache
+
+Cache handling for feeds specified by `<PackageSource>` in Central Feed Management is separate from feeds specified in NuGet.config.
+
+## Cache Organization
+
+In order to enforce the security rules of [Feed Scoping](/feeds/grouping-and-source-pinning/#feed-scoping), packages downloaded using Central Feed Management will be stored in a separate cache.
+
+For projects with Central Feed Management enabled, the global package cache will not be utilized.
+
+The Central Feed Management cache organizes a sub-cache for every feed.
+
+#### Cache Storage Locations
```

Comment:
Due to the way VM are created:
* OS Disk
* Data Disk

We use `EnvVar` to change these cache location today on the CI, because the `Os Disk` is small and we can't predict when it will be critical.
This might help scenario like `Azure Disk` being mounted to re-use cache <== not sure if this is a good idea ^^